### PR TITLE
Sync with cds-runtime

### PR DESCRIPTION
**Pull request by MikhailGoncharov**

### CODE REVIEW

**Review by danjoa — APPROVED** [review]

File: test/cds.ql.test.js [hunk 1]
```diff
@@ -309,7 +309,7 @@ describe('cds.ql → cqn', () => {
                 'and',
                 { ref: ['args'] },
                 'in',
-                { val: args },
+                { list: [{ val: 'foo' }, { val: 'bar' }, { val: 3 }] },
```
Comment (danjoa) [inline test/cds.ql.test.js:312]: Shouldn't that be this?:
```suggestion
                { list: [{ val: 'foo' }, { val: "'bar'" }, { val: 3 }] },
```

**Review by danjoa — CHANGES_REQUESTED** [review]

File: test/cds.ql.test.js (hunk 1 above)
Comment (MikhailGoncharov) [reply test/cds.ql.test.js:312]: > Shouldn't that be this?:

I found that we trim single quotes in case fo Fluid API. Should it be in sync?

Comment (danjoa) [reply test/cds.ql.test.js:312]: I don't think so → we should also double check why we trim quotes in fluent API; sounds wrong

Comment (MikhailGoncharov) [reply test/cds.ql.test.js]: Updated. Also, provided a bit more info per email.

**Review by danjoa — APPROVED** [review]

**Pull request merged**